test/support/test_helpers.ex

```diff
@@ -0,0 +1,28 @@
+defmodule Twitter.TestHelpers do
+  alias Twitter.{
+    Accounts
+  }
+
+  alias Twitter.{
+    Twits
+  }
+
+  def user_fixture(attrs \\ %{}) do
+    {:ok, user} =
+      attrs
+      |> Enum.into(%{
+        name: "Some User",
+        username: "user#{System.unique_integer([:positive])}",
+        password: attrs[:password] || "supersecret",
+        email: "email"
+      })
```

Comment:
why not a map.merge or something more specific? 

you may want defaults set for one map and override what comes in through the `attrs` using map merge. `attrs[:password] || "supersecret"`will be unnecessary if you make the default `supersecret` and override with the merge.  Maybe I am missing something about the way its used.

Reply:
Ok, I will research that

Reply:
`Map.merge` & `Enum.into` would work almost the same, but the difference will be in the function signature: `Enum.into` allows your function to work with keywords as well as maps. I think this is what was recommended in the phoenix book and is probably better.
雑談ですが、performance-wise `Enum.into` is optimized to perform a `Map.merge` when merging a map into a map. Not that it is a concern here, especially in tests, but just saying `Enum.into` can be really nice to use when you want to work with both keywords and maps.

Reply:
Jean san, thank you for your comment. This function is indeed from the textbook. I have also tried to make a default map values of which can be overwritten but the attrs came in a list format, so I instead of taking time to troubleshoot it (not that I can't), I decided to keep this function.